content/docs/self-hosted-runners.md

```diff
@@ -361,6 +406,41 @@ for obtaining these keys.
 ☝️ **Note** The same credentials can also be used for
 [configuring cloud storage](/doc/cml-with-dvc#cloud-storage-provider-credentials).
 
+**Example:** IAM permissions needed for `cml runner`:
```

Comment:
low-level enough to be moved to `/content/docs/ref/runner.md` & cross-referenced?